src/control/cmd/daos/pool.go

```diff
@@ -224,9 +224,8 @@ type poolCmd struct {
 
 type poolQueryCmd struct {
 	poolBaseCmd
-	ShowEnabledRanks  bool `short:"e" long:"show-enabled" description:"Show engine unique identifiers (ranks) which are enabled"`
-	ShowDisabledRanks bool `short:"b" long:"show-disabled" description:"Show engine unique identifiers (ranks) which are disabled"`
-	HealthOnly        bool `short:"t" long:"health-only" description:"Only perform pool health related queries"`
+	ShowEnabledRanks bool `short:"e" long:"show-enabled" description:"Show engine unique identifiers (ranks) which are enabled"`
```

Comment:
Conceptually this feels strange to me. Is the default to _only_ show disabled ranks?

Reply:
Yep, it was a request from @mjmac and @Michael-Hennecke to always have the disabled ranks as this information is really important for the admin.  From my side, I also agree on this as displayed this information can help the admin to realize that some nodes are unavailable, and thus some admin tasks are needed.

Reply:
OK, that makes sense. Thanks for the further context. :)